app/main/views/marketplace.py

```diff
@@ -73,3 +73,30 @@ def get_brief_by_id(framework_slug, brief_id):
         brief=brief,
         content=brief_content
     )
+
+
+@main.route('/<framework_slug>/opportunities')
+def list_opportunities(framework_slug):
+    page = process_page(request.args.get('page', default=u"1"))
```

Comment:
`request.args.get` also accepts `type=int` (http://werkzeug.pocoo.org/docs/0.11/datastructures/#werkzeug.datastructures.TypeConversionDict.get), so there's no need for a separate conversion function